### PR TITLE
Update secure_change_url to use env var

### PR DESCRIPTION
**Pull request by Rhian**
### Description of change

This simplifies the url construction to use the APPLICANTS_APP_HOST env var

### CODE REVIEW

File: app/models/planning_application.rb [hunk 1]
```diff
@@ -219,11 +219,8 @@ def full_address
   end
 
   def secure_change_url(application_id, secure_token)
-    case ENV["DOMAIN"]
-    when "bops-staging.services"
-      "http://#{local_authority.subdomain}.bops-applicants-staging.services/change_requests?planning_application_id=#{application_id}&change_access_id=#{secure_token}"
-    when "bops.services"
-      "http://#{local_authority.subdomain}.bops-applicants.services/change_requests?planning_application_id=#{application_id}&change_access_id=#{secure_token}"
+    if ENV["DOMAIN"]
```
Comment (james) [inline app/models/planning_application.rb:222]: Why do we have this `if` here?

Comment (Rhian) [reply app/models/planning_application.rb:222]: To differentiate between http and https - ENV["DOMAIN"] is only set on Prod and Staging. Ideally we'd replace this with a ENV["PROTOCOL"] but that would mean another Ansible update on both Prod and Staging that we would have to coordinate.

Comment (james) [reply app/models/planning_application.rb:222]: What about `if Rails.env.production?`

Comment (Rhian) [reply app/models/planning_application.rb:222]: Much better, thanks! Will do now.

**Review by james — APPROVED** [review]

**Pull request merged**